sklearn/feature_selection/tests/test_sequential.py

```diff
@@ -136,12 +222,25 @@ def test_pipeline_support():
 
     # pipeline in SFS
     pipe = make_pipeline(StandardScaler(), LinearRegression())
-    sfs = SequentialFeatureSelector(pipe, cv=2)
+    sfs = SequentialFeatureSelector(pipe, n_features_to_select="auto", cv=2)
     sfs.fit(X, y)
     sfs.transform(X)
 
     # SFS in pipeline
-    sfs = SequentialFeatureSelector(LinearRegression(), cv=2)
+    sfs = SequentialFeatureSelector(
+        LinearRegression(), n_features_to_select="auto", cv=2
+    )
     pipe = make_pipeline(StandardScaler(), sfs)
     pipe.fit(X, y)
     pipe.transform(X)
+
+
+# FIXME : to be removed in 1.2
+def test_raise_deprecation_warning():
+    """Check that we raise a deprecation warning with `n_features_to_select`."""
+    n_samples, n_features = 50, 3
+    X, y = make_regression(n_samples, n_features, random_state=0)
+
+    warn_msg = "Leaving ...."
```

Comment:
It would be better to put the beginning of the string indeed

```suggestion
    warn_msg = "Leaving n_features_to_select to None is deprecated"
```